scripts/shared/lib/utils

```diff
@@ -52,6 +52,19 @@ function run_parallel() {
     done
 }
 
+# Run cluster commands sequentially.
+# 1st argument is the numbers of the clusters to run for, supports "1 2 3" or "{1..3}" for range
+# 2nd argument is the command to execute, which will have the $cluster variable set.
+# 3rd argument and so forth get passed to the command.
+function run_sequential() {
+    local clusters cmnd
+    clusters=$(eval echo "$1")
+    cmnd=$2
+    for i in ${clusters}; do
+        ( with_context "cluster${i}" "$cmnd" "${@:3}" | sed "s/^/[cluster${i}] /" )
```

Comment:
```suggestion
        with_context "cluster${i}" "$cmnd" "${@:3}" | sed "s/^/[cluster${i}] /"
```

Subshell isn't really needed here, it was more important for the parallel execution

Reply:
Ack, thanks @mkolesnik